torch/testing/_internal/common_nn.py

```diff
@@ -106,14 +106,18 @@ def get_weight(m):
         cpp_constructor_args='torch::nn::LinearOptions(10, 8)',
         input_size=(4, 10),
         reference_fn=lambda i, p, _: torch.mm(i, p[0].t()) + p[1].view(1, -1).expand(4, 8),
+        with_tf32=True,
+        tf32_precision=0.005,
```

Comment:
Which flag controls the behavior of linear?

Reply:
It should be the cuBLAS flag, in the cuBLAS PR, I didn't look at `test_torch.py` because there are so many cuDNN failing tests.